QUICKSTART.md

```diff
@@ -81,5 +81,10 @@ The binding tools need the following to operate:
 
 In addition, the tools can operator on separate `.swiftmodule` and `.dylib` files. This can be handled by using a `-M` argument for the `.swiftmodule` and `-L` for the library. It’s easier to use the `.framework` directory and a single `-C` argument.
 
-
+## Building and Running Samples
+Samples source code is in the `samples` directory, but are will not build and run there.
+Instead you need to make a packaged build, from the root of binding-tools-for-swift execute the command:
+	`make package`
```

Comment:
If this was intended to show up on a different line, you need empty lines before and after:

```suggestion

	`make package`

```

Otherwise it's just one long sentence: https://github.com/xamarin/binding-tools-for-swift/blob/5af3823b94234f9c148707e1b8f70c94e749caf5/QUICKSTART.md#building-and-running-samples